deeplake/core/vectorstore/vector_search/vector_search.py

```diff
@@ -18,9 +18,9 @@
 
 def search(
     k: Optional[int],
-    distance_metric: str,
     exec_option: str,
     deeplake_dataset: DeepLakeDataset,
+    distance_metric: str = "L2",
```

Comment:
@sounakr Cal you pls clarify the docstring since it no longed matches the code behavior. It should be clear how the distance metric selected when it is specified as `None`.